doc/installation.rst

```diff
@@ -33,8 +37,9 @@ to `pysal/spaghetti`_, you can contribute to the giddy development.
 
 |
 
-.. _3.5: https://docs.python.org/3.5/
 .. _3.6: https://docs.python.org/3.6/
+.. _3.7: https://docs.python.org/3.6/
```

Comment:
should this be https://docs.python.org/3.7/ ?

Reply:
yes, good catch